src/styles/Button.js

```diff
@@ -1,29 +1,33 @@
 export const ButtonSettings = {
-    componentVariants: {
-        color: {
-            prop: 'colorVariant',
-            variants: {
-                'solid-primary': {
-                    backgroundColor: 'primary.main',
-                    // backgroundColor: 'orangered',
+  componentVariants: {
+    color: {
+      prop: "colorVariant",
+      variants: {
+        "solid-primary": {
+          backgroundColor: "primary.main",
+          // backgroundColor: 'orangered',
 
-                    boxShadow: 'none',
-                    '&:hover': {
-                        // backgroundColor: 'primary.dark',
-                        backgroundColor: '#FF0000',
-                    }
-                },
-                'solid-secondary': {
-                    backgroundColor: 'transparent',
-                    boxShadow: 'none'
-                },
-                'outline-primary': {
-                    boxShadow: 'none'
-                },
-                'outline-secondary': {
-                    boxShadow: 'none'
-                }
-            },
+          boxShadow: "none",
+          "&:hover": {
+            // backgroundColor: 'primary.dark',
+            backgroundColor: "#FF0000",
+          },
         },
+        "solid-secondary": {
```

Comment:
the real changes here, added better colors for "loading more" button

below before & after

<img width="1431" alt="Screenshot 2023-06-01 at 11 52 28" src="https://github.com/maxio-com/vue-self-service-example/assets/91472483/565a3878-06f7-4fc9-9116-0901a8b2f9de">
<img width="1468" alt="Screenshot 2023-06-01 at 11 52 48" src="https://github.com/maxio-com/vue-self-service-example/assets/91472483/a0b9fe90-0266-41bd-b772-654c2f1fe1fe">